helpers/authentication.js

```diff
@@ -0,0 +1,26 @@
+export const authTokenKey = 'Authentication';
+
+export const getToken = () => {
+  const jsonAuthTokens = localStorage.getItem(authTokenKey);
+  if (!jsonAuthTokens) {
+    return;
+  }
+
+  const tokens = JSON.parse(jsonAuthTokens);
+
+  return tokens;
+};
+
+export const setToken = (tokenType, accessToken, refreshToken) => {
+  const tokens = {
```

Comment:
```suggestion
  const token = {
```

Should we name it in a singular way? 
I understand the idea of plural due to the presence of 2 tokens (access and refresh), but I think we can keep singular for the following reasons:
- to be consistant with the function name `setToken` (it is not `setTokens`)
- to ease distinction between array and object (we would keep the plural form for array variables and singular for object variables) ;-)

Reply:
👍  to the idea of singular object variable names, refactored across the PR in d001d237faad2946d3f13cd8a0e2207485664da4